### PR TITLE
Dta primal

### PR DESCRIPTION
**Pull request by happypig95**
Not sure if we want to merge everything here into master since some are specifically created for CAV artifact requirements. Let me know if you think some file should not appear in the master.

### CODE REVIEW

File: bench/tasks/bitprecise.set [hunk 1]
```diff
@@ -32,7 +32,6 @@ bitprecise/Singapore-2.yml
 bitprecise/Stockholm-1.yml
 bitprecise/SyntaxSupportPointer01-1.yml
 bitprecise/SyntaxSupportPointer01-2.yml
-bitprecise/SyntaxSupportPointer01-3.yml
```
Comment (zkincaid) [inline bench/tasks/bitprecise.set:35]: Was this benchmark proved unsafe by uautomizer?

File: bench/tools/cpachecker.py [hunk 2]
```diff
@@ -0,0 +1,231 @@
+# This file is part of BenchExec, a framework for reliable benchmarking:
```
Comment (zkincaid) [inline bench/tools/cpachecker.py]: Is this file necessary?  BenchExec distribution should already have this file.

File: srk/src/terminationDTA.ml [hunk 3]
```diff
@@ -440,14 +412,58 @@ module XSeq = struct
...
+        (mk_mod srk t (mk_real srk (QQ.of_int divisor)))
+        (mk_zero srk)
     in
     Periodic.map mk_divides dividend_xseqs
 
   let seq_of_notdivides_atom srk zz_divisor dividend_vec exp_poly abstraction =
     seq_of_divides_atom srk zz_divisor dividend_vec exp_poly abstraction
     |> seq_not srk
   
+  let omega_dom_basis_mat dim omega_dom_mat =
```
Comment (zkincaid) [inline srk/src/terminationDTA.ml:343]: Need some comments describing the specification of the functions in this block

**Review by zkincaid — COMMENTED** [review]
This pull request has conflicts in srk/src/terminationDTA.ml -- please pull in the latest changes first.

File: bench/tasks/bitprecise.set (hunk 1 above)
Comment (happypig95) [reply bench/tasks/bitprecise.set:35]: Yes this is the problematic one we decided to exclude by the submission time.

File: bench/tools/cpachecker.py (hunk 2 above)
Comment (happypig95) [reply bench/tools/cpachecker.py]: I think the BencExec's version should be good.
